_toc.yml

```
@@ -0,0 +1,17 @@
+file: README
+sections:
+- file: governance
+- file: newsubprojects
+- file: papers
+- file: people
+- file: projectlicense
+- file: trademarks
+- file: conduct/code_of_conduct
```

How about this order, which more closely aligns with the readme order, and I think makes a little more sense to read?

```suggestion
- file: governance
- file: conduct/code_of_conduct
- file: people
- file: newsubprojects
- file: papers
- file: trademarks
- file: projectlicense
```

---

or perhaps people should come before code of conduct?

---

ah good idea - I was trying not to take any strong opinion about structure and just use the ordering based on file names, but I agree that the "Table of Contents" section already defines this structure so I think we can just use that without saying we're changing the governance documents themselves